docs/osf-institutions-sso-via-saml.md

```diff
@@ -37,19 +41,24 @@ COS offers a Service Provider (SP) based on [SAML 2.0](https://docs.oasis-open.o
   * Production: https://accounts.osf.io/Shibboleth.sso/Metadata
   * Test and/or staging: https://accounts.test.osf.io/Shibboleth.sso/Metadata
 
-* Ensure that your IT administrators are releasing the three required pieces of information listed below and inform COS of the attributes you use for each of them.
-  * Unique identifier for the user (e.g. `eppn`)
-  * User's institutional email (e.g. `mail`)
-  * User's full name (e.g. `displayName` or **a pair of** `givenName` and `sn`)
+* Ensure that your IT administrators are releasing the three required pieces of information listed below. Optional ones are highly recommended if possible. Inform COS of the attributes you use for each of them.
+  * Required
+    * Unique identifier for the user (e.g. `eppn`)
+    * User's institutional email (e.g. `mail`)
+    * User's full name (e.g. `displayName`)
+  * Optional
+    * User's first and last name (e.g. a pair of `givenName` and `sn`)
+    * User's department at your institution (e.g. `eduPersonOrgUnitDN` or `eduPersonPrimaryOrgUnitDN`)
+    * User's relationship(s) (e.g. student, faculty, staff, alum, etc.) to the institution (e.g. `eduPersonAffiliation` or `eduPersonPrimaryAffiliation`)
 
+* Provide COS with the metadata of your IdP server(s). URL is preferred over a physical file.
```

Comment:
Rephrase this as the following:

> Provide COS with IdP metadata for your test / stage (if available) and prod servers. A URL to the metadata is preferred over an XML file so that our SP server can periodically reload and refresh the metadata.